Sources/SwiftParser/generated/Parser+TokenSpecSet.swift

```diff
@@ -0,0 +1,2027 @@
+//// Automatically generated by generate-swiftsyntax
+//// Do not edit directly!
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+@_spi(RawSyntax) import SwiftSyntax
+
+extension Parser {
```

Comment:
I don’t think there’s a need to namespace this to parser. The `TokenSpecSet` in TokenSpecSet.swift also aren’t namespaces.